src/components/TransactionFailText/index.tsx

```diff
@@ -41,7 +41,7 @@ export const TransactionFailText = ({
   if (isExecution) {
     errorMessage =
       threshold && threshold > 1
-        ? `To save gas costs, cancel this transaction`
+        ? `To save gas costs, reject this transaction`
```

Comment:
shouldn't is say `To save gas, reject..`

Reply:
Probably, but not sure. @lukasschor?

Reply:
I would even say "to save fees, ..." but we should check that this is aligned with other places where we might use similar wording.